app/src/main/java/com/github/se/cyrcle/model/user/UserRepository.kt

```diff
@@ -10,11 +10,11 @@ interface UserRepository {
   fun onSignIn(onSuccess: () -> Unit)
 
   /**
-   * Get a new unique identifier for a parking
+   * Get a a unique identifier for a user
    *
-   * @return a new unique identifier for a parking
+   * @return A UID linked to the user's signed in account
    */
```

Comment:
If you think this is a better name you should change it across all repository and viewmodel for consitency, I don't really think which name we use matter, but it should be the same accross the project